src/library/Menu/Menu.js

```diff
@@ -97,7 +62,9 @@ export default class Menu extends PureComponent<MenuProps> {
     const { props: itemProps } = props;
     const { index, item } = itemProps;
 
-    if (item.divider) {
+    if (item.group) {
+      return <MenuGroup key={index} title={item.text} />;
```

Comment:
Did you consider making the "item" signature for a group `{ group: true, title: 'Group Title' }`? While using `text` is consistent with the other items, `title` both matches the `children`-based API (`<MenuGroup title="Group Title" />`) and helps to differentiate group titles from menu items. Imo, at least.

Reply:
Updated.